distro/src/avail/Avail.avail/Unit Testing.avail/Tools.avail

```diff
@@ -126,15 +126,19 @@ Public explicit class "failed-requirement exception" extends exception;
  *         If {@param "predicate"} is false.
  * @category "Unit Testing"
  */
-Public method "Require:_" is [predicate : boolean |] : ⊤;
+Public method "Require:_" is [predicate : boolean |] : ⊤
+	styled by assertion styler;
```

Comment:
Outdent.  And below.

Reply:
There's only one tab there. I just verified it. Nothing to do.